optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -647,12 +662,20 @@ def best_trial(self) -> optuna.trial.FrozenTrial:
 
 
 class LightGBMTuner(LightGBMBaseTuner):
-    """Hyperparameter-tuning with Optuna for LightGBM.
+    """Hyperparameter tuner for LightGBM.
+
+    It optimizes the following hypeperparameters in stepwise manner:
+    ``lambda_l1``, ``lambda_l2``, ``num_leaves``, ``feature_fraction``, ``bagging_fraction``,
+    ``bagging_freq`` and ``min_child_samples``.
+
+    You can find the detailed algorithm and benchmark results in `this blog article <https://mediu
+    m.com/optuna/lightgbm-tuner-new-optuna-integration-for-hyperparameter-optimization-8b7095e9925
+    8>`_ by `Kohei Ozaki <https://www.kaggle.com/confirm>`_, who is a Kaggle Grandmaster.
```

Comment:
A matter of style. Sounds a bit more formal which is fine here. You can ignore it if you want.
```suggestion
    8>`_ by `Kohei Ozaki <https://www.kaggle.com/confirm>`_, a Kaggle Grandmaster.
```